### DISCUSSION TITLE
Not working with typescript

### DISCUSSION THREAD

**Discussion by VaZark** [ivan-dalmet/formiz#119] [Q&A]
I love the ease of use that Formiz promises. However, I am not able to get it working with typescript. 

Do you have any recipes with TS?  Does the library support it ?

**Comment by yoannfleurydev** [comment]
Hi @VaZark , you can check https://github.com/BearStudio/start-ui-web that is working fine with Formiz and TypeScript (and many other libs)

**Reply by yoannfleurydev** [reply]
If you do not find any solution using the configuration from 🚀 Start UI [web], feel free to update your post with screenshots or snippets that might help us debugging Formiz using TypeScript.

**Comment by HugoPerard** [comment]
Hey @VaZark !
We are working on a Formiz types override on Start-UI that allow you to type field value and otherProps, and form values. You can check [this PR](https://github.com/BearStudio/start-ui-web/pull/312) for more details.
